mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/MapInstance.java

```diff
@@ -386,7 +386,7 @@ public void onDestroy() {
 
```

Comment:
Did the Unit tests run with this change?
Were new Unit tests added for this capability?
Will Capability tester be updated?